frappe/core/doctype/user/user.js

```diff
@@ -1,4 +1,14 @@
 frappe.ui.form.on("User", {
+	setup: function (frm) {
+		frm.set_query("default_workspace", () => {
+			return {
+				filters: {
+					parent_page: ["is", "not set"],
+					for_user: ["in", [null, frappe.session.user]],
```

Comment:
```suggestion
					for_user: ["in", [null, frappe.session.user]],
					title: ["!=", "Welcome Workspace"],
```

Reply:
Why `parent_page: ["is", "not set"]`?

Reply:
Loading a "child" workspace was broken when this feature was initially developed a long time ago, not anymore.